mentorship/2020/gsod/README.md

```diff
@@ -43,6 +43,55 @@ You’d get to join GraphQL working groups to give updates, write about your wor
 
 **Mentor:** [The Guild](http://github.com/the-guild-org), Others
 
+### 3. Revamp `graphql-js` docs (medium/hard)
+
+**Review and update existing docs and complete it with tutorials and FAQ**
+
+There is a [documentation for graphql-js](https://graphql.org/graphql-js/), but it is severely outdated because it was handwritten for the first public release four years ago. It was rarely updated since.
+
+The idea is to synchronize it with the source code by generating it from JSDoc comments. GraphQL.js maintainers will handle docs generation so you can focus on:
+
+  - improving the quality of existing JSDoc comments
+  - cover missing functions/arguments
+  - add examples
+  
+The next step would be to implement documentation best practices as Introduction, Tutorials, FAQ.
+
+You will be working with GraphQL.js maintainers who will be ready to answer your questions and provide technical expertise.
+
+We expect you to be able to write code snippets in JS/TS, understand and explain complex concepts as lexer, parser, AST, etc.
+
+**Expected outcomes:** Full and up to date documentation for `graphql-js`.
+
+**Skills required/preferred:** knowledge of GraphQL, JavaScript/TypeScript
```

Comment:
, AST experience preferred (but not required?)

Reply:
@acao I think it's actually helpful if a person will learn these concepts in a process since it will result in docs that are approachable by beginners. It's not that hard to document `validate` API functions it's way harder to write a tutorial that guides a beginner (in ASTs and parsers) through writing custom validation rule.